horizomer/utils/tests/test_tree.py

```diff
@@ -39,116 +40,232 @@ def tearDown(self):
         # but in the future there will be
         rmtree(self.working_dir)
 
+    def test_support(self):
+        """Test getting support value of a node."""
+        # test nodes with support alone as label
+        tree = TreeNode.read(['((a,b)75,(c,d)90);'])
+        node1, node2 = tree.children
+        self.assertEqual(support(node1), 75.0)
+        self.assertEqual(support(node2), 90.0)
+
+        # test nodes with support and branch length
+        tree = TreeNode.read(['((a,b)0.85:1.23,(c,d)0.95:4.56);'])
+        node1, node2 = tree.children
+        self.assertEqual(support(node1), 0.85)
+        self.assertEqual(support(node2), 0.95)
+
+        # test nodes with support and extra label (not a common scenario but
+        # can happen)
+        tree = TreeNode.read(['((a,b)\'80:X\',(c,d)\'60:Y\');'])
```

Comment:
This will also work in python
```
tree = TreeNode.read(["((a,b)'80:X',(c,d)'60:Y');"])
```
So no need for escape characters (unless you want them).

Reply:
I am trying to uniformly use `'` instead of `"` in all str cases. How does this sound?